resize map command is broken

Works with the last release, broken on bleed.

`mono OpenRA.Utility.exe ra --resize-map /home/dr/.openra/maps/ra/\{DEV_VERSION\}/oasis.oramap 86 86`

 > Unhandled Exception:
System.IO.FileNotFoundException: Unable to find parent package for /home/dr/.openra/maps/ra/{DEV_VERSION}/oasis.oramap
  at OpenRA.FileSystem.ZipFile..ctor (IReadOnlyFileSystem context, System.String filename) <0x416a3b50 + 0x002bf> in <filename unknown>:0 
  at OpenRA.FileSystem.FileSystem.OpenPackage (System.String filename) <0x4167c280 + 0x00157> in <filename unknown>:0 
  at OpenRA.Mods.Common.UtilityCommands.ResizeMapCommand.Run (OpenRA.ModData modData, System.String[] args) <0x416a3360 + 0x00077> in <filename unknown>:0 
  at OpenRA.Utility.Program.Main (System.String[] args) <0x4163f9e0 + 0x005f0> in <filename unknown>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: System.IO.FileNotFoundException: Unable to find parent package for /home/dr/.openra/maps/ra/{DEV_VERSION}/oasis.oramap
  at OpenRA.FileSystem.ZipFile..ctor (IReadOnlyFileSystem context, System.String filename) <0x416a3b50 + 0x002bf> in <filename unknown>:0 
  at OpenRA.FileSystem.FileSystem.OpenPackage (System.String filename) <0x4167c280 + 0x00157> in <filename unknown>:0 
  at OpenRA.Mods.Common.UtilityCommands.ResizeMapCommand.Run (OpenRA.ModData modData, System.String[] args) <0x416a3360 + 0x00077> in <filename unknown>:0 
  at OpenRA.Utility.Program.Main (System.String[] args) <0x4163f9e0 + 0x005f0> in <filename unknown>:0 

[Logs_utility.zip](https://github.com/OpenRA/OpenRA/files/354025/Logs_utility.zip)

--- a/OpenRA.Mods.Common/UtilityCommands/ExtractMapRules.cs
+++ b/OpenRA.Mods.Common/UtilityCommands/ExtractMapRules.cs
@@ -54,7 +54,7 @@ namespace OpenRA.Mods.Common.UtilityCommands
 		{
 			Game.ModData = modData;
 
-			var map = new Map(modData, modData.ModFiles.OpenPackage(args[1]));
+			var map = new Map(modData, modData.ModFiles.OpenPackage(args[1], new Folder(".")));
 			MergeAndPrint(map, "Rules", map.RuleDefinitions);
 			MergeAndPrint(map, "Sequences", map.SequenceDefinitions);
 			MergeAndPrint(map, "VoxelSequences", map.VoxelSequenceDefinitions);

--- a/OpenRA.Mods.Common/UtilityCommands/GetMapHashCommand.cs
+++ b/OpenRA.Mods.Common/UtilityCommands/GetMapHashCommand.cs
@@ -10,6 +10,7 @@
 #endregion
 
 using System;
+using OpenRA.FileSystem;
 
 namespace OpenRA.Mods.Common.UtilityCommands
 {
@@ -25,7 +26,7 @@ namespace OpenRA.Mods.Common.UtilityCommands
 		[Desc("MAPFILE", "Generate hash of specified oramap file.")]
 		public void Run(ModData modData, string[] args)
 		{
-			using (var package = modData.ModFiles.OpenPackage(args[1]))
+			using (var package = modData.ModFiles.OpenPackage(args[1], new Folder(".")))
 				Console.WriteLine(Map.ComputeUID(package));
 		}
 	}

--- a/OpenRA.Mods.Common/UtilityCommands/OutputActorMiniYamlCommand.cs
+++ b/OpenRA.Mods.Common/UtilityCommands/OutputActorMiniYamlCommand.cs
@@ -1,6 +1,7 @@
 using System;
 using System.IO;
 using System.Linq;
+using OpenRA.FileSystem;
 
 namespace OpenRA.Mods.Common.UtilityCommands
 {
@@ -27,7 +28,7 @@ namespace OpenRA.Mods.Common.UtilityCommands
 				try
 				{
 					mapPath = args[2];
-					map = new Map(modData, modData.ModFiles.OpenPackage(mapPath));
+					map = new Map(modData, modData.ModFiles.OpenPackage(mapPath, new Folder(".")));
 				}
 				catch (InvalidDataException)
 				{

--- a/OpenRA.Mods.Common/UtilityCommands/ResizeMapCommand.cs
+++ b/OpenRA.Mods.Common/UtilityCommands/ResizeMapCommand.cs
@@ -49,7 +49,7 @@ namespace OpenRA.Mods.Common.UtilityCommands
 		public void Run(ModData modData, string[] args)
 		{
 			Game.ModData = modData;
-			map = new Map(modData, modData.ModFiles.OpenPackage(args[1]));
+			map = new Map(modData, modData.ModFiles.OpenPackage(args[1], new Folder(".")));
 			Console.WriteLine("Resizing map {0} from {1} to {2},{3}", map.Title, map.MapSize, width, height);
 			map.Resize(width, height);
 

